tests/Integration/Behaviour/Features/Context/StoreFeatureContext.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+namespace Tests\Integration\Behaviour\Features\Context;
+
+use Behat\Gherkin\Node\TableNode;
+use Country;
+use Store;
+use Tests\Integration\Behaviour\Features\Context\Util\PrimitiveUtils;
+
+class StoreFeatureContext extends AbstractPrestaShopFeatureContext
+{
+    /**
+     * @var int default lang id from configs
+     */
+    private $defaultLangId;
+
+    public function __construct()
+    {
+        $configuration = CommonFeatureContext::getContainer()->get('prestashop.adapter.legacy.configuration');
+        $this->defaultLangId = (int) $configuration->get('PS_LANG_DEFAULT');
```

Comment:
We already have a `AbstractDomainFeatureContext::getDefaultLangId` helper function to do this, however you can't extend it here since this is not a "Domain context" so it's fine for


Just keep in mind when you create the `AddStoreCommand` (if you planned on doing it after), that you will need to migrate this behat endpoint into the store domain context and at this point you'll be able to use `$this->getDefaultLangId()`

That said, you will probably need to handle localized values so actually you won't even need this info anymore, the data from feature files will have to be a localized array